documentation/jetty/modules/programming-guide/pages/server/http.adoc

```diff
@@ -1040,6 +1063,8 @@ Refer to the `EventsHandler` link:{javadoc-url}/org/eclipse/jetty/server/handler
 
 `QoSHandler` allows web applications to limit the number of concurrent requests, therefore implementing a quality of service (QoS) mechanism for end users.
```

Comment:
Is it really limiting "concurrent requests" ?  It does not limit the number of request, just the number that are actively being handled. It is limiting threads running concurrent requests or something to that effect.

We should also say why this is a good thing

Reply:
Done.